go.mod

```diff
@@ -26,11 +27,13 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
-	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/iavl v0.14.0
-	github.com/tendermint/tendermint v0.33.7
+	github.com/tendermint/go-amino v0.16.0
```

Comment:
Better not to update this, right?
I am happy with the patch version upgrades below, but 0.15 -> 0.16 may have a breaking change (even in serialized format)

Reply:
Reverted this :ballot_box_with_check: